packages/radio-button/src/RadioButton.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { ControlBehavior } from "@hig/behaviors";
+import { generateId } from "@hig/utils";
+
+import RadioButtonPresenter from "./presenters/RadioButtonPresenter";
+
+export default function RadioButton(props) {
+  const {
+    checked,
+    name,
+    defaultChecked,
+    disabled,
+    onBlur,
+    onChange,
+    onClick,
+    onFocus,
+    onMouseDown,
+    onMouseEnter,
+    onMouseLeave,
+    onMouseUp,
+    ...otherProps
+  } = props;
+
+  const id = props.id || generateId("radio-button");
```

Comment:
`id`, and `name` might be better as part of `...otherProps` and we probably don't want to generate an id for this.

Reply:
I guess generateId was only necessary previously for the label integration. Now that labels are gone we probably don't need to generate them.